scripts/check_python_licenses.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Deffo the right set-tings

`set -o errexit` and `set -o nounset` are arguably more readable

Reply:
It's common enough in bash-world and fairly easy to google. And there's always the man pages!